docs/csharp/quick-starts/interpolated-strings.yml

```diff
@@ -0,0 +1,176 @@
+       public static void Main()
+       {
+          var item = new Vegetable("eggplant");
+          var date = DateTime.Now;
+          var price = 1.99m;
+          var unit = Unit.item;
+          Console.WriteLine($"On {date}, the price of {item} was {price} per {unit}.");
+       }
+    }
+    ```
+    
+    Note that once of the interpolated expressions includes the `item` object in the result string that's displayed to the console, and in this case the string "eggplant" is inserted into the result string. That's because, when the type of an interpolated expression is not a string, the C# compiler does the following:
+
+    - If the interpolated expression is `null`, the interpolated expression returns an empty string ("", or <xref:System.String.Empty?displayProperty=nameWithType>).
+
+    - If the interpolated expression is not `null`, the `ToString` method of the type of the interpolated expression is called. You can test this by commenting out the definition of the `Vegetable.ToString` method in the example by putting a comment symbol (`\\`) in front of it. In the output, the string "eggplant" is replaced by the fully qualified type name (the namespace defined by the C# REPL along with the type name), which is the default behavior of the <xref:System.Object.ToString?displayProperty=nameWithType> method.   
+
+    In the output from this example, the date is too precise (the price of eggplant does not vary by the second), and the price value doesn't indicate a unit of currency. In the next quick start, you'll learn how to fix those issues by controlling the format of strings returned by interpolated expressions.
+
+    > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Control the formatting of interpolated expressions
+  durationInMinutes: 5
+  content: |
+    In the previous lesson, two strings that were poorly formated were inserted into the result string. One was a date and time value for which only the date was appropriate. The second was a price that did not indicate its unit of currency. Both issues are easy to address. An interpolated expression can include *format strings* that control the formatting of particular types. Run the following code in the interactive window to format the date and time and the price.
+
+    ```csharp
+    public class Vegetable
+    {
+       public Vegetable(string name) => Name = name;
+   
+       public string Name { get; }
+   
+       public override string ToString() => Name;
+    }
+
+    public class Example
+    {
+       public enum Unit { item, pound, ounce, dozen };
+   
+       public static void Main()
+       {
+          var item = new Vegetable("eggplant");
+          var date = DateTime.Now;
+          var price = 1.99m;
+          var unit = Unit.item;
+          Console.WriteLine($"On {date:d}, the price of {item} was {price:C2} per {unit}.");
```

Comment:
I'd display only this line (the change to Console.WriteLine) in this step. Then, I'd change the text from "run the following code..." to "modify the call to Console.WriteLine from the previous example to include the format specifier for the `date` and `price` fields as shown in the following line:"

When you press the "next" button on the previous step, all the code from the previous example is already in the interactive window. Changing the instructions encourages the reader to make the change instead of copy / paste the entire sample.